### DISCUSSION TITLE
Add new beatoraja play skin to list?

### DISCUSSION THREAD

**Discussion by catcraze777** [wcko87/beatoraja-english-guide#46] [Guide discussion]
Hey, as the title says I was wondering if it would be possible to add a new play skin to the beatoraja skin list? I recently finished developing a play skin I'm calling CC-Play and was wanting to get this out there into others' hands. You can find my repository with it here: https://github.com/catcraze777/ccplay 

Here's a pic of it in case it looks interesting.
![java_05_23-32-45](https://github.com/wcko87/beatoraja-english-guide/assets/52767118/3b9036d5-21db-4520-badb-b98a328e7579)

**Comment by wcko87** [comment]
Just added your skin to the list. Thanks!

**Comment by catcraze777** [comment]
I've added support for 14-key and 10-key if you want to update the information again.

**Comment by wcko87** [comment]
updated.
